api/v3/Mailing.php

```diff
@@ -613,19 +614,24 @@ function civicrm_api3_mailing_send_test($params) {
   if (!array_key_exists('test_group', $params) && !array_key_exists('test_email', $params)) {
     throw new API_Exception("Mandatory key(s) missing from params array: test_group and/or test_email field are required");
   }
-  civicrm_api3_verify_mandatory($params,
-    'CRM_Mailing_DAO_MailingJob',
-    array('mailing_id'),
-    FALSE
-  );
+  $testEmailParams = [
+    'mailing_id' => $params['mailing_id'],
```

Comment:
@eileenmcnaughton your 100% mailing_id is always getting passed as  param right?

Reply:
@seamuslee001 it's a required field - ie I moved a verify_mandatory call to a spec declaration

Reply:
ok happy with that then